daemon/src/common/String.h

```diff
@@ -234,6 +234,18 @@ namespace Str {
             return c;
     }
 
+    /*
+     * Converts a hexadecimal character to the value of the digit it represents.
+     * Pre: cisxdigit(ch)
+     */
+    inline CONSTEXPR_FUNCTION int gethex( char ch )
```

Comment:
Probably said earlier but functions that mimc c functions should still get a uppercase first letter (and you can remove the padding in the parenthesis)